.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   build:
     working_directory: ~/project/sam-design-system
     docker:
-      - image: circleci/node:10.14.0-browsers
+      - image: circleci/node:12.22.7-browsers
```

Comment:
Updated this based on [this stackoverflow thread](https://stackoverflow.com/questions/67550901/object-fromentries-is-not-a-function-error-when-using-chakra-ui-and-next-js) circleCi build was failing due to no having that function available in Node 10, resolved when updated to Node 12